webview/src/App.svelte

```diff
@@ -0,0 +1,511 @@
+  import Panzoom from '@panzoom/panzoom';
+  import { onMount } from 'svelte';
+  import panIcon from './assets/pan.svg';
+  import zoominIcon from './assets/zoom-in.svg';
+  import zoomoutIcon from './assets/zoom-out.svg';
+  import layouts from '@mermaid-chart/layout-elk';
+  import { vscode } from './utility/vscode'
+
+  let diagramContent: string = `flowchart TD
+      %% Nodes
+          A("fab:fa-youtube Starter Guide")
+          B("fab:fa-youtube Make Flowchart")
+          n1@{ icon: "fa:gem", pos: "b", h: 24}
+          C("fa:fa-book-open Learn More")
+          D{"Use the editor"}
+          n2(Many shapes)@{ shape: delay}
+          E(fa:fa-shapes Visual Editor)
+          F("fa:fa-chevron-up Add node in toolbar")
+          G("fa:fa-comment-dots AI chat")
+          H("fa:fa-arrow-left Open AI in side menu")
+          I("fa:fa-code Text")
+          J(fa:fa-arrow-left Type Mermaid syntax)
+
+      %% Edge connections between nodes
+          A --> B --> C --> n1 & D & n2
+          D -- Build and Design --> E --> F
+          D -- Use AI --> G --> H
+          D -- Mermaid js --> I --> J
+
+      %% Individual node styling. Try the visual editor toolbar for easier styling!
+          style E color:#FFFFFF, fill:#AA00FF, stroke:#AA00FF
+          style G color:#FFFFFF, stroke:#00C853, fill:#00C853
+          style I color:#FFFFFF, stroke:#2962FF, fill:#2962FF
+
+      %% You can add notes with two "%" signs in a row!`;
+ 
+  let errorMessage = "";
+  let isToggled = true;
+  $: zoomLevel = 100;
+  let panzoomInstance: ReturnType<typeof Panzoom> | null = null;
+  let panEnabled = false;
+  let isErrorOccured= false;
+  let theme: "default" | "base" | "dark" | "forest" | "neutral" | "neo" | "neo-dark" | "mc" | "null" = "neo"; 
+  $: sidebarBackgroundColor = theme === "dark" || theme === "neo-dark" ? "#4d4d4d" : "white";
+  $: iconBackgroundColor = theme === "dark" || theme === "neo-dark" ? "#4d4d4d" : "white";
+  $: svgColor = theme === "neo-dark" || theme === "dark" ? "white" : "#2329D6";
+  $: shadowColor =
+    theme === "dark" || theme === "neo-dark" ? "#6b6b6b" : "#A3BDFF";
+
+
+  async function initializeMermaid() {
+    try {
+      mermaid.registerLayoutLoaders(layouts);
+      mermaid.registerIconPacks([
+        {
+          name: 'fa',
+          loader: () => import('@iconify-json/fa6-regular').then((m) => m.icons),
+        },
+        {
+          name: 'aws',
+          loader: () => import('@mermaid-chart/icons-aws').then((m) => m.icons),
+        },
+        {
+          name: 'azure',
+          loader: () => import('@mermaid-chart/icons-azure').then((m) => m.icons),
+        },
+        {
+          name: 'gcp',
+          loader: () => import('@mermaid-chart/icons-gcp').then((m) => m.icons),
+        },
+      ]);
+      let config = {
+        startOnLoad: false,
+        suppressErrorRendering: true,
+        theme: theme
+      }
+      await mermaid.initialize(config);
```

Comment:
Move the config directly inside the initialize function, so it's typed properly.